### DISCUSSION TITLE
Using capture text in particular position in templeter

### DISCUSSION THREAD

**Discussion by link-ding** [chhoumann/quickadd#153] [Q&A]
I have an action which caputre the text and create a new file by using templater. I want to put the caputre text in the particular postion in the templete. Is there any way I can do this?

**Comment by chhoumann** [comment]
Coming back to this one very late - sorry about that.

You don't need Templater for this - QuickAdd runs its own format syntax on the template when it creates the file. Put a value token in the template exactly where the captured text should go:

1. Make a Template choice pointing at your template file.
2. In the template, place `{{VALUE:my text}}` at the exact position where the text belongs:

```
# {{VALUE:title}}

Some fixed content...

{{VALUE:my text}}

More fixed content...
```

When the choice runs, QuickAdd prompts for each token and inserts your text right at that spot in the new file.

If you're instead capturing into notes that already exist, use a Capture choice with the **After line...** write position ("Insert after") - that drops the captured text under a specific heading or line rather than at the top/bottom.

Docs: https://quickadd.obsidian.guide/docs/FormatSyntax#named-value

Closing as resolved.
